src/components/routes/Host/ListingForm/ListingForm.container.ts

```diff
@@ -96,4 +95,26 @@ const ListingFormContainer = styled.section`
   }
 `;
 
-export default ListingFormContainer;
\ No newline at end of file
+const ListingFormTabletContainer = styled(ListingFormMobileContainer)`
+  @media (min-width: 768px) {
+    & > .bee-general-wrapper {
+      form {
+        padding: 40px 56px 96px 24px;
+      }
+    }
+  }
+`;
+
+const ListingFormDesktopContainer = styled(ListingFormTabletContainer)`
+  @media (min-width: 1025px) {
+    & > .bee-general-wrapper {
+      width: 976px;
+      form {
+        padding-left: 0;
```

Comment:
for conformity, i would use `padding: 40px 56px 96px 0;`

Reply:
if this is a consistent padding for all form containers, can it be a variable somewhere or just applied to `form` generally?

Reply:
I think the `padding` was for the layout of this particular form -- though that would be something good to keep in mind a `<BeeFormContainer>` that has all the layouts/styles for a basic form.